samples/net/zperf/src/zperf_tcp_receiver.c

```diff
@@ -143,21 +145,36 @@ void zperf_tcp_receiver_init(const struct shell *shell, int port)
 			return;
 		}
 
-		ret = zperf_get_ipv4_addr(shell, MY_IP4ADDR,
-					  &in4_addr_my->sin_addr);
-		if (ret < 0) {
-			shell_fprintf(shell, SHELL_WARNING,
-				      "Unable to set IPv4\n");
-			return;
+		if (MY_IP4ADDR && strlen((const char *)MY_IP4ADDR)) {
+			/* Use Setting IP */
+			ret = zperf_get_ipv4_addr(shell, MY_IP4ADDR,
+					in4_addr);
+			if (ret < 0) {
+				shell_fprintf(shell, SHELL_WARNING,
+						"Unable to set IPv4\n");
```

Comment:
indendation seems to be slighly off here. Please also check other shell_fprintf() commands as the indentation is off there too.